ExpandRegion.py

```diff
@@ -81,18 +82,25 @@ def run(self, edit, language="", undo=False, debug=True):
         if debug:
           print("startIndex: {0}, endIndex: {1}, type: {2}".format(result["start"], result["end"], result["type"]))
       else:
-        # if there is no result, keep the current region
-        new_regions.append(region)
-
-    # replace the selections with the new regions
-    view.sel().clear()
-    for sel in new_regions:
-      view.sel().add(sel)
-
-    settings = sublime.load_settings("ExpandRegion.sublime-settings")
-    do_force_enable_soft_undo = settings.get("force_soft_undo_integration")
-    if do_force_enable_soft_undo:
-      _force_enable_soft_undo(view, edit, new_regions)
+        # # if there is no result, keep the current region
+        # new_regions.append(region)
```

Comment:
What is the effect when this is commended out?

Also: remove dead code instead of comment.

Reply:
When this is commented out we cannot recreate the same regions, however when falling back to Sublime Text built-in command, we do not need to recreate regions. Also, I am sure why before the regions were being recreated when there was no new regions. It seems redundant to recreate them when no new regions are being added.